posts/2024-02-15-Windows-7.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Updated baseline standards for Windows targets"
+author: Chris Denton on behalf of the Compiler Team
+---
+
+The minimum requirements for Tier 1 toolchains targeting Windows will increase with the 1.77 release (scheduled for March 21, 2024).
```

Comment:
This should be 1.78 right? Unless we're backporting the doc change 🙂

Reply:
I mean, couldn't we? It's early in the cycle. But, yeah, I don't mind changing it.